cmd/testrunner.go

```diff
@@ -7,26 +7,35 @@ import (
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
+	"github.com/elastic/elastic-package/internal/cobraext"
 	"github.com/elastic/elastic-package/internal/packages"
 	"github.com/elastic/elastic-package/internal/testrunner"
 	"github.com/elastic/elastic-package/internal/testrunner/system"
 )
 
+const (
```

Comment:
I can see these flags (and their descriptions) being used in multiple commands, especially the `dataset` flag. In any case it might be good to keep all possible flags and their descriptions in a single file to make it easier to achieve consistency in naming and language. WDYT about moving them to something like `internal/cobraext/const.go`?

Reply:
Fixed